build/common.sh

```diff
@@ -704,6 +704,7 @@ function kube::build::copy_output() {
     --filter='+ zz_generated.*' \
     --filter='+ generated.proto' \
     --filter='+ *.pb.go' \
+    --filter='+ types.go' \
```

Comment:
Have I ever told you the story about how I hate this rsync stuff, and MacOS users should suck it up and get a real OS for building?